robozonky-strategy-natural/src/main/antlr4/imports/MarketplaceFilters.g4

```diff
@@ -198,11 +202,18 @@ primaryMarketplaceFilterCondition returns [MarketplaceFilterCondition result]:
 ;
 
 secondaryMarketplaceFilterCondition returns [MarketplaceFilterCondition result]:
-    c1=regionCondition { $result = $c1.result; }
-    | c2=ratingCondition { $result = $c2.result; }
+    c2=ratingCondition { $result = $c2.result; }
     | c3=incomeCondition { $result = $c3.result; }
     | c4=purposeCondition { $result = $c4.result; }
-    | c5=storyCondition { $result = $c5.result; }
+    | c6=termCondition { $result = $c6.result; }
+    | c8=interestCondition { $result = $c8.result; }
+    | c9=relativeTermCondition { $result = $c9.result; }
+    | c10=elapsedTermCondition { $result = $c10.result; }
+    | c11=elapsedRelativeTermCondition { $result = $c11.result; }
+;
+
+sellMarketplaceFilterCondition returns [MarketplaceFilterCondition result]:
+    c2=ratingCondition { $result = $c2.result; }
```

Comment:
@jhrcek The changes relevant to the web tool are all in this file. I removed some joint filters and some secondary filters. I also created a whole new set of filters for selling, as those no longer match the secondary filters.